src/css/index.scss

```diff
@@ -0,0 +1,418 @@
+$primary_color: #ddd;
```

Comment:
Good use of variables here, but could you try to extract the exact colors from your browser web inspector?